src/LibPQ.jl

```diff
@@ -21,42 +28,41 @@ using Memento: Memento, getlogger, warn, info, error, debug
 using OffsetArrays
 using TimeZones
 
-const Parameter = Union{String, Missing}
+const Parameter = Union{String,Missing}
 const LOGGER = getlogger(@__MODULE__)
 
 function __init__()
     INTERVAL_REGEX[] = _interval_regex()
-    Memento.register(LOGGER)
+    return Memento.register(LOGGER)
 end
 
 # Docstring template for types using DocStringExtensions
-@template TYPES =
-    """
-        $(TYPEDEF)
+@template TYPES = """
+                      $(TYPEDEF)
 
-    $(DOCSTRING)
+                  $(DOCSTRING)
 
-    ## Fields:
+                  ## Fields:
 
-    $(TYPEDFIELDS)
-    """
+                  $(TYPEDFIELDS)
+                  """
```

Comment:
These lines should all remain indented at the previous level (moving the opening quotes to the previous line is good). 

This is covered in BlueStyle in the [Whitespace](https://github.com/invenia/BlueStyle/#whitespace) section, scroll down to triple quotes.